roles/consul/templates/nginx-consul.service.j2

```diff
@@ -19,10 +19,13 @@ ExecStart=/usr/bin/docker run \
     --net=host \
     --privileged=true \
     -v /etc/nginx/ssl:/etc/nginx/ssl:ro \
+    -v /etc/nginx/templates:/etc/nginx/templates:ro \
     --env-file=/etc/default/nginx-consul.env \
     {{ consul_nginx_image }}:{{ consul_nginx_image_tag }}  
 
 ExecStop=/usr/bin/docker kill nginx-consul
 
+ExecReload=/usr/bin/docker kill -HUP nginx-consul
```

Comment:
HUP is caught by the `launch.sh` script. When the HUP is caught, the consul-template configurations are generated and consul-template is reloaded. It could be any signal I suppose but HUP is generally used for reloading configuration.